src/app-layer-ftp.h

```diff
@@ -115,6 +50,11 @@ typedef struct FTPString_ {
     TAILQ_ENTRY(FTPString_) next;
 } FTPString;
 
+typedef struct FtpCommandInfo_ {
+    uint8_t command_code;
+    uint8_t command_index;
```

Comment:
These 2 fields feel redundant...

Reply:
If you use a rust enum for `FTP_COMMAND_UNKNOWN` and such, you may define a stringer function on it

The derive `EnumStringU8` gives it  to you, but not with lowercase I am afraid